### PR TITLE
UI :- Remove ellipses component

### LINKED ISSUE #7404: UI : Remove ellipsis component
Remove ellipsis component

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue : Closes #7404
- Remove ellipses component
- Use ellipses as per content type

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx [hunk 1]
```diff
@@ -113,9 +112,11 @@ export const EntityListWithAntd: FunctionComponent<AntdEntityListProp> = ({
                       className="tw-text-grey-body hover:tw-text-primary-hover hover:tw-underline"
                       title={getEntityName(item as unknown as EntityReference)}
                       type="text">
-                      <Ellipses className="tw-w-48 tw-text-left">
+                      <Typography.Text
+                        className="ant-typography-ellipsis-custom tw-w-48 tw-text-left"
+                        ellipsis={{ tooltip: true }}>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx:117]: Can we use our classes for width and text alignment?

File: openmetadata-ui/src/main/resources/ui/src/components/app-bar/Appbar.tsx [hunk 2]
```diff
@@ -185,13 +188,11 @@ const Appbar: React.FC = (): JSX.Element => {
...
           to={getUserPath(currentUser?.name as string)}>
           {' '}
-          <Ellipses
-            tooltip
-            className="tw-font-medium tw-cursor-pointer"
-            rows={1}
-            style={{ color: '#7147E8' }}>
+          <Typography.Paragraph
+            className="ant-typography-ellipsis-custom font-medium cursor-pointer text-primary"
+            ellipsis={{ rows: 1, tooltip: { name } }}>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/app-bar/Appbar.tsx:193]:
```suggestion
            ellipsis={{ rows: 1, tooltip: true }}>
```

File: openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx [hunk 3]
```diff
@@ -489,14 +488,13 @@ const TagsPage = () => {
...
-                  className="tag-category label-category tw-self-center tw-w-32"
+                <Typography.Paragraph
+                  className="ant-typography-ellipsis-custom tag-category label-category tw-self-center tw-w-32"
                   data-testid="tag-name"
-                  rows={1}>
+                  ellipsis={{ rows: 1, tooltip: true }}>
                   {getEntityName(category as unknown as EntityReference)}
-                </Ellipses>
-
+                  Gupta
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx:496]: Can we not check in the dummy code?

**Review by Sachin-chaurasiya — COMMENTED** [review]
@Ashish8689 , Few comments.

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  All comments are addressed!

File: openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx]: @Ashish8689 , Can you make these changes to all the files in this PR?

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 4]
```diff
@@ -56,17 +59,57 @@
...
+.w-24 {
+  width: 6rem;
+}
 .w-32 {
   width: 8rem;
 }
 .w-48 {
   width: 12rem;
 }
+.w-11-12 {
```
[the remark this answers is not in the document]
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:71]: @Sachin-chaurasiya  css is not excepting forward Slash in creating class.  can we use `.w-11\12` for this secenario ?

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:71]: @Ashish8689 , No let's keep it like this only.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**